src/js/components/Diagram/doc.js

```diff
@@ -18,6 +18,16 @@ export const doc = Diagram => {
     connections: PropTypes.arrayOf(
       PropTypes.shape({
         anchor: PropTypes.oneOf(['center', 'vertical', 'horizontal']),
+        animation: PropTypes.oneOfType([
+          PropTypes.bool,
+          PropTypes.oneOf(['pulse', 'draw']),
+          PropTypes.shape({
+            type: PropTypes.oneOf(['pulse', 'draw']),
+            delay: PropTypes.number,
+            duration: PropTypes.number,
+            size: PropTypes.string,
```

Comment:
What does size affect? What are possible values?

Reply:
Added the possible values as well as specified in the code comment that pulse is the only animation currently that uses size